BLSQ/iaso
# WC2-431: Decipher how to aggregate ETL based on current tableau

What problem is this PR solving? Explain here in one sentence.

Related JIRA tickets : [WC2-431](https://bluesquare.atlassian.net/browse/WC2-431)

## Self proofreading checklist



## Changes

Aggregated data by period(month and year) and org unit.

## How to test

From the local instance with wfp coda database, run ETL script to generate data for:
- South Sudan: `docker compose run iaso manage etl_ssd `
- Nigeria: `docker compose run iaso manage etl_ng`

Then login on django admin and check on [Monthly statistics table](http://localhost:8081/admin/wfp/monthlystatistics/). You should see data aggregated by period and org unit. You can also filter the data per account(country).

## Print screen / video


[Administration-de-Wfp-Iaso.webm](https://github.com/user-attachments/assets/cd50e1a7-867b-481e-9464-c40e447cfcbe)



## Follow the Conventional Commits specification

The **merge message** of a pull request must follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.

This convention helps to automatically generate release notes.

Use lowercase for consistency.

[Example](https://github.com/BLSQ/iaso/commit/8b8d7d3064138c1e57878f17b4eb922516ab0112):

```
fix: empty instance pop up

Refs: IA-3665
```

Note that the Jira reference is preceded by a _line break_.

Both the line break and the Jira reference are entered in the _Add an optional extended description…_ field.


[WC2-431]: https://bluesquare.atlassian.net/browse/WC2-431?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
